src/compiler/comments.ts

```diff
@@ -5,12 +5,13 @@ namespace ts {
     export interface CommentWriter {
         reset(): void;
         setSourceFile(sourceFile: SourceFile): void;
-        getLeadingComments(range: Node, shouldSkipCommentsForNodeCallback?: (node: Node) => boolean): CommentRange[];
+        getLeadingComments(range: Node, shouldSkipCommentsForNodeCallback?: (node: Node) => boolean, getCustomTextRangeForNodeCallback?: (node: Node) => TextRange): CommentRange[];

```

Comment:
Could you rename "getCustomTextRangeForNodeCallback" to "getCommentRange"? as the latter is the one that is used as call back? This is to just make it easier to find the implementation